### DISCUSSION TITLE
UI improvement

### DISCUSSION THREAD

**Discussion by chguw** [KrisCris/Palworld-Pal-Editor#22] [Ideas]
**Is your feature request related to a problem? Please describe.**
This is an amazing tool with exactly the features I need, but I was a little confused by the interface when I first used it. For example, the "Apply" button, which appears as a green checkmark surrounded by a light green background, means "checked" to me at first glance. And the selected item in the PLAYER LIST panel on the left appears gray while the unselected ones remain colored, which is counterintuitive and was the opposite of the PAL LIST.

**Describe the solution you'd like**
Perhaps a more common and consistent interface would improve usability.

**Comment by KrisCris** [comment]
What icon do you think works better?

What color would you suggest? FYI they are just the `:disabled` colors that I selected.
